policyengine_canada/parameters/gov/provinces/ab/tax/income/credits/credits_return/upper_dependant_income_threshold.yaml

```diff
@@ -0,0 +1,16 @@
+description: Alberta caregiver amount phases out for dependants above this income threshold.
```

Comment:
```suggestion
description: Alberta phases its caregiver amount out for dependants above this income threshold.
```

Reply:
But does it phase out or is it an eligibility threshold?

The variable logic seems to suggest that it is just a limit and there is no phaseout - can you double check